R/deployApp.R

```diff
@@ -345,6 +337,10 @@ deployApp <- function(appDir = getwd(),
   } else {
     displayStatus(paste0(capitalize(assetTypeName), " deployment failed ",
                          "with error: ", response$error, "\n"))
+
+    # a client can still require config with some deployment failures
+    checkForConfig(client, application, showURL, quiet)
```

Comment:
If the application can't be deployed for e.g. a permissions issue, what will this do?